File: frontend/src/app/pages/auth/dto/login-response.dto.ts
export interface LoginResponseDto {
    authenticateResult: boolean;
    authToken: string;
    accessTokenExpireDate: string;
    refreshToken?: string | null;
    refreshTokenExpireDate?: string | null;
    defaultRoute: string | null;
    userStatus: string | null;
    permissions: string[];
    aktifKurumId?: number | null;
    kurumIds?: number[];
    kurumAdminKurumIds?: number[];
    isKurumAdmin?: boolean;
    isSuperAdmin?: boolean;
}

File: platform/TOD.Platform.Security/Auth/DTO/LoginResponseDto.cs
namespace TOD.Platform.Security.Auth.DTO;

public class LoginResponseDto
{
    public bool AuthenticateResult { get; set; }

    public string AuthToken { get; set; } = string.Empty;

    public DateTime AccessTokenExpireDate { get; set; }

    public string RefreshToken { get; set; } = string.Empty;

    public DateTime? RefreshTokenExpireDate { get; set; }

    public string? UserStatus { get; set; }
}
